integration_test/dapp_tests/hardhat.config.js

```diff
@@ -43,5 +52,14 @@ module.exports = {
         count: 1
       },
     },
+    devnetFastTrack: {
```

Comment:
how is this config different than `devnet`?

Reply:
Actually they are the same, removed and introduced another flag to the command -f. if passed with devnet it will run against already deployed contract, if passed with seicluster, it will create a config file where it records the contract addresses on the first iteration and run for the subsequent times based on this config file.